src/system/SearchModule/Block/SearchBlock.php

```diff
@@ -94,7 +135,7 @@ public function getFormOptions()
         }
         // remove disabled
         foreach ($searchModules as $displayName => $moduleName) {
-            if ((bool) $this->getVar('disable_' . $moduleName, true)) {
+            if ((bool) $this->getVar('disable_' . $moduleName, false)) {
```

Comment:
BC break?
I'd guess that a default value of `true` is more reasonable because it does not immediately add new modules to the search (which may not be desired).